src/abacus-ts/websocket/lib/indexerWebsocket.ts

```diff
@@ -163,25 +170,31 @@ export class IndexerWebsocket {
     if (message.message.startsWith('Internal error, could not fetch data for subscription: ')) {
       const maybeChannel = message.channel;
       const maybeId = message.id;
-      if (maybeChannel != null && maybeId != null && maybeChannel.startsWith('v4_')) {
-        const channelAndId = `${maybeChannel}${CHANNEL_ID_SAFE_DIVIDER}${maybeId}`;
+      if (maybeChannel != null && maybeChannel.startsWith('v4_')) {
+        const channelAndId = `${maybeChannel}${CHANNEL_ID_SAFE_DIVIDER}${maybeId ?? NO_ID_SPECIAL_STRING_ID}`;
         const lastRefresh = this.lastRetryTimeMsByChannelAndId[channelAndId] ?? 0;
         if (Date.now() - lastRefresh > CHANNEL_RETRY_COOLDOWN_MS) {
           this.lastRetryTimeMsByChannelAndId[channelAndId] = Date.now();
           this._refreshSub(maybeChannel, maybeId);
-          logAbacusTsInfo(
-            'IndexerWebsocket',
-            'error fetching data for channel, refetching',
+          logAbacusTsInfo('IndexerWebsocket', 'error fetching data for channel, refetching', {
```

Comment:
not sure if this would make sense, but maybe have abstract to `logAbacusTsIndexerWs` within this file to avoid having to add expected parameters in each call?